src/coreclr/jit/jit.h

```diff
@@ -440,6 +440,7 @@ class GlobalJitOptions
 #define MEASURE_MEM_ALLOC 1 // Collect memory allocation stats.
 #define LOOP_HOIST_STATS 1  // Collect loop hoisting stats.
 #define TRACK_LSRA_STATS 1  // Collect LSRA stats
+#define TRACK_ENREG_STATS 1 // Collect enregistration stats
```

Comment:
Set
```
#define TRACK_ENREG_STATS 0
```
in the `#else` clause, like the other cases?

Reply:
I was considering it but the contract here is that you then will be able to set `#define TRACK_ENREG_STATS 1` and get data in the release. It is not the case here, in order to make `TRACK_ENREG_STATS 1` work in the release I would need to replace many `DEBUGARG` with something like 
```
#if defined(DEBUG) || (TRACK_ENREG_STATS  == 1)
#define DEBUG_OR_TACK_ENREG_STATS(arg) , arg
#endif
```
and it looked to me as overkill, especially because for this mode we don't expect different results from the release and checked like for others.